src/lib/esnext.arraybuffer.d.ts

```diff
@@ -0,0 +1,58 @@
+interface ArrayBuffer {
+	/**
+	 * Read-only. The maximum length that this ArrayBuffer can be resized to (in bytes).
+	 */
+	readonly maxByteLength: number;
+
+	/**
+	 * Read-only. Whether this ArrayBuffer can be resized or not.
```

Comment:
```suggestion
	 * Returns true if this ArrayBuffer can be resized.
```

Actually, my comments about growable apply; I'd prefer to delete this comment.